### DISCUSSION TITLE
Improving Voice -> Text for FSNotes using Whisper?

### DISCUSSION THREAD

**Discussion by ReessKennedy** [glushchenko/fsnotes#1731] [Ideas]
The default voice to text for FS Notes doesn't seem that great. OpenAI's Whisper Voice to text is amazingly accurate! What about using it in FS Notes now that it is open source? That would be really cool.

**Comment by gingerbeardman** [comment]
Which voice-to-text feature?

How are you doing it in FSNotes? Screenshots or screen recording would help.

**Reply by ReessKennedy** [reply]
Right! Realizing that I am actually just using iOS voice to text that is outside of FS Notes and that FS Notes does not have a built in feature like this. Maybe just an idea ... I guess there are keyboard replacement tools that would also override the default iOS solution that may be much better I should look into more. Still, OpenAI's open sourcing of Whisper is pretty neat and presents some opp for open source app developers that want to build cool stuff with it.

**Reply by gingerbeardman** [reply]
Google keyboard also has its own voice to text.

And let's see what iOS 18 brings.
